webextensions/api/devtools.json

```
@@ -51,16 +51,100 @@
               }
             }
           },
+          "Resource": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/inspectedWindow/resource",
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "16"
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            },
+            "getContent": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": true,
+                    "notes": "Only base64 is supported for encoded content."
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "16"
+                  },
+                  "safari_ios": {
+                    "version_added": false
+                  }
+                }
+              }
+            },
+            "setContent": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": true
```

Per [Chrome's docs](https://developer.chrome.com/docs/extensions/reference/api/devtools/inspectedWindow#method-Resource-setContent), this method only supports text content.
```suggestion
                    "version_added": true,
                    "notes": "Only resources with type \"text\" are supported."
```